### DISCUSSION TITLE
sound capture

### DISCUSSION THREAD

**Discussion by thelvyn777** [awawa-dev/HyperHDR#87] [Q&A] [answered]
Hello,

I have got a working installation on a Pi4 with a HDMI/USB capure device capturing from a computer runing Kodi
Got no problem with Video or Visual Effect but I cannot make the "Music Effect" working with music coming from the same HDMI/USB capure device.

I have tested all the "Sound capture device" in the "Effects Menu".
Did I forget something ?

**Comment by awawa-dev** [comment, marked as answer]
Hi
You must make sure that the sound stream is coming to your USB capturing device (in format that your grabber can capture). It's a particular setup depended thing. All I could help is described in the manual: 
https://hyperhdr.blogspot.com/2021/04/how-to-set-up-hyperhdr-part-i-basic.html#chapter6
Start from verifying your sound capture device setup setup & logs.

**Reply by thelvyn777** [reply]
Prob is coming from my Denon AVR.
Sound is not send though HDMI Output AND Speakers at the same time,  need to choose between them.
Will try with ARC.

**Reply by awawa-dev** [reply]
Yes, I also have Denon and same problem. On the other hand Yamaha works.
You could also try to capture HDMI signal before Denon if you have a splitter or a grabber with a loop.
